src/ImageSharp/Primitives/DenseMatrix{T}.cs

```diff
@@ -109,7 +109,7 @@ public DenseMatrix(T[,] data)
         /// <returns>The <see typeparam="T"/> at the specified position.</returns>
         public ref T this[int row, int column]
         {
-            [MethodImpl(InliningOptions.ShortMethod)]
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Why is it `AggressiveInlining` here and not the internal `ShortMethod`? Was this intentional or did you just forget to change this back after your initial experiments?

Reply:
Intentional. `ShortMethod` was initially designed for certain jpeg profiling but it kinda spread. I don't think we need it any more wit the tooling we have available.

Reply:
Ah, got it. Well we might want to revert that in another PR then 😄